steps/start/postcode-checker/BranchForEnglandOrWales.js

```diff
@@ -0,0 +1,64 @@
+const { branch, goTo } = require('@hmcts/one-per-page');
+const request = require('request');
+const config = require('config');
+const { Logger } = require('@hmcts/nodejs-logging');
+
+const postcodeCountryLookupUrl = config.get('postcodeChecker.url');
+const postcodeCountryLookupToken = config.get('postcodeChecker.token');
+const postcodeCountryLookupAllowedCountries = config.get('postcodeChecker.allowedCountries');
+
+const logger = Logger.getLogger('PostcodeChecker.js');
+
+class BranchForEnglandOrWales {
+  constructor(postcode, englandOrWalesStep, otherStep, errorStep) {
+    this.postcode = postcode;
+    this.englandOrWalesStep = englandOrWalesStep;
+    this.otherStep = otherStep;
+    this.errorStep = errorStep;
+  }
+
+  isEnglandOrWalesPostcode() {
+    const postcode = this.postcode;
+    return new Promise((resolve, reject) => {
+      request.get({
+        headers: {
+          Authorization: `Token ${postcodeCountryLookupToken}`
+        },
+        url: `${postcodeCountryLookupUrl}/${postcode}`
+      }, (error, resp, body) => {
+        if (error) {
+          return reject(error);
+        }
+
+        const okStatusCode = 200;
```

Comment:
We're using a module called `http-status-codes` for stuff like this: 
https://github.com/hmcts/submit-your-appeal/blob/3ea6035a5e3ba91b2e062218a25224ebddbefce4/steps/check-your-appeal/CheckYourAppeal.js#L11

So here you would to `resp.statusCode !== HttpStatus.OK`

Reply:
Done